packager/hls/base/media_playlist_unittest.cc

```diff
@@ -299,6 +356,41 @@ TEST_F(MediaPlaylistMultiSegmentTest, SetTargetDuration) {
   ASSERT_FILE_STREQ(kMemoryFilePath, kExpectedOutput);
 }
 
+TEST_F(MediaPlaylistMultiSegmentTest, WriteToFileWithTagExtXProgramDateTime) {
```

Comment:
I think you can remove these new tests except the first one as they don't add more test coverage.

Reply:
I suggest also not to remove the last test because it adds case when the time stamp changes across key frames.

Reply:
Sure.

Reply:
Done.